scripts/fires/wfigs_data_test.py

```diff
@@ -0,0 +1,79 @@
+"""This script has the tests for wfigs_data.py.
+It imports two test datasets and checks if the processed output matches the
+expected output.
+"""
+import unittest
+import os
+import pandas as pd
+import sys
+from datetime import datetime
+
+module_dir_ = os.path.dirname(__file__)
+sys.path.insert(0, module_dir_)
+import wfigs_data
+
+
+def _GetTestPath(relative_path):
+    return os.path.join(module_dir_, relative_path)
+
+
+def _GetEpochMillis(x):
+    if not pd.isna(x):
+        utc_time = datetime.strptime(x, "%Y/%m/%d %H:%M:%S")
+        return (utc_time - datetime(1970, 1, 1)).total_seconds() * 1000
+    return None
+
+
+def _GetDatetime(x):
+    if not pd.isna(x):
+        return datetime.strptime(x, "%Y-%m-%d %H:%M:%S")
+    return None
+
+
+class PreprocessDataTest(unittest.TestCase):
+    """This class has the method required to test preprocess_data script."""
+
+    def test_ProcessDF(self):
```

Comment:
the test shouldn't include processing code -- the processing code should be factored in a way such that we can just pass in the input data, and compare the results with expected.

Reply:
Done